src/test/acceptance/features/privacy-notice.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Privacy notice page
```

Comment:
Should this be *privacy policy*? Or is that the agreed language? *privacy notice* sounds like a popup or banner (subjective I guess).

Reply:
That's what it is called in the prototype - i.e. in the content from Kate